### PR TITLE
Fix health check docs

### LINKED ISSUE #8491: [DOC] Monitoring Server Health
**Description:**
There are several issues such as the following regarding the documentation [1] on Monitoring Server Health:

1. In Step 1 part 6 there is no file with the name: health-check.config.xml in the path: <API_HOME>/features/org.wso2.carbon.healthcheck.server_1.0.0.
But there is a file with the name: health-check-config.xml in the path: <HEALTH_CHECK_HOME>/features/org.wso2.carbon.healthcheck.server.feature/resources/health-check-config.xml.

2. The 'sample configuration' has a duplicate entry of [carbon_health_check.health_checker.data_source_health_checker] which is not valid for the deployment.toml file.
```

[carbon_health_check] 
enable = true

[carbon_health_check.health_checker.data_source_health_checker]
enable = true
pool_usage_limit_percentage = "80"
order = "97"

[carbon_health_check.health_checker.data_source_health_checker.properties]
'monitored.datasources' = "jdbc/WSO2CarbonDB,jdbc/WSO2AM_DB,jdbc/SHARED_DB"

[carbon_health_check.health_checker.data_source_health_checker]
enable =false
order = 98

[carbon_health_check.health_checker.super_tenant_health_checker.properties]
'monitored.user.stores' = "primary" 

[[health_checker]] 
name = "customChecker" 
order = "87"
[health_checker.properties] 
property1 = "property-1-value" 
property2 = "property-2-value" 


```
[1] https://apim.docs.wso2.com/en/latest/administer/logging-and-monitoring/monitoring/monitoring-server-health/#monitoring-server-health

**Suggested Labels:**

4.3.0

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by msm1992**
## Purpose

Fixes #8491

### CODE REVIEW

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md [hunk 1]
```diff
@@ -239,17 +239,14 @@ Follow the instructions below to add a custom Health Checker:
 
     [carbon_health_check.health_checker.data_source_health_checker.properties]
     'monitored.datasources' = "jdbc/WSO2CarbonDB,jdbc/WSO2AM_DB,jdbc/SHARED_DB"
-    
-    [carbon_health_check.health_checker.data_source_health_checker]
-    enable =false
-    order = 98
```
Comment (ashera96) [inline en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md:245]: Shouldn't this be the configuration for super tenant health checker? Just like the configurations that come before this are for `carbon_health_check.health_checker.data_source_health_checker` and `carbon_health_check.health_checker.data_source_health_checker.properties`; I believe this should be a typo. We should change this from `[carbon_health_check.health_checker.data_source_health_checker]` to `[carbon_health_check.health_checker.super_tenant_health_checker]`

```
    [carbon_health_check.health_checker.super_tenant_health_checker]
    enable =false
    order = 98
```

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ This section guides you through deploying the Carbon Health Check components in
 
 5. Copy the webapp `api#health-check#v1.0.war`, which is found in the `<HEALTH_CHECK_HOME>/components/org.wso2.carbon.healthcheck.api.endpoint/target/` directory and paste it in the `<API-M_HOME>/repository/deployment/server/webapps` directory.
 
-6. Copy the `health-check.config.xml` configuration file found in the `<API_HOME>/features/org.wso2.carbon.healthcheck.server_1.0.0` directory to your `<PRODUCT_HOME>/repository/conf/` directory.
+6. Copy the `health-check-config.xml` configuration file found in the `<HEALTH_CHECK_HOME>/features/org.wso2.carbon.healthcheck.server.feature/resources` directory to your `<PRODUCT_HOME>/repository/conf/` directory.
```
Comment (ashera96) [inline en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md:35]: Will changing the `health-check-config.xml` work fine for this? Isn't it the j2 or a toml file that we should be editing?

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md (hunk 1 above)
Comment (msm1992) [reply en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md]: fixed with https://github.com/wso2/docs-apim/pull/8697/commits/f4b07aa1cacf3578d48860f553da4affdc2ace6e

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md (hunk 2 above)
Comment (ashera96) [reply en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md:35]: Ack, marking this as resolved.

**Review by ashera96 — APPROVED** [review]

**Pull request merged**